### ISSUE TITLE
New bug between 0.7.4 and 0.7.5: SQS only allows one queue at a time

### ISSUE THREAD

**Issue by katharosada** [localstack/localstack#324]
Creating and listing two queues on docker version 0.7.4:
```
$ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name "myqueue1" --profile localstack
{
    "QueueUrl": "http://localhost:4576/123456789012/myqueue1"
}
$ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name "myqueue2" --profile localstack
{
    "QueueUrl": "http://localhost:4576/123456789012/myqueue2"
}
$ aws sqs list-queues --endpoint-url=http://localhost:4576 --profile localstack
{
    "QueueUrls": [
        "http://localhost:4576/123456789012/myqueue1",
        "http://localhost:4576/123456789012/myqueue2"
    ]
}
```

When I run the same on 0.7.5 or higher:
```
$ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name "myqueue1" --profile localstack
{
    "QueueUrl": "http://localhost:4576/queue/myqueue1"
}
$ aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name "myqueue2" --profile localstack
{
    "QueueUrl": "http://localhost:4576/queue/myqueue2"
}
$ aws sqs list-queues --endpoint-url=http://localhost:4576 --profile localstack
{
    "QueueUrls": [
        "http://localhost:4576/queue/myqueue2"
    ]
}
```

The account ID is no longer listed in the url, and only one queue is ever kept. Always the most recently created queue.

**Comment by fabriciogpp** [comment]
+1

**Comment by whummer** [comment]
Thanks for reporting, this should be fixed in #342 . Please re-open this ticket if the problem persists.
